connectors/storages.mongodb/src/storage.js

```diff
@@ -30,13 +30,19 @@ class Storage extends Connector {
 
   async get (query) {
     const { criteria, options } = translate(query)
+
+    console.debug('Database query', { method: 'findOne', criteria, options })
```

Comment:
I believe this is some _reinventing the wheel_ style implementation. We should use MongoDB Node.js driver feature(s) for that - https://www.mongodb.com/docs/drivers/node/v5.5/fundamentals/logging/